### ISSUE TITLE
Alpine images need nsswitch.conf for DNS

### ISSUE THREAD

**Issue by dnwe** [ibmruntimes/ci.docker#1]
As per sgerrand/alpine-pkg-glibc#4 the alpine images need an nsswitch.conf file else Java won't take account of /etc/hosts entries such as those created by Docker --add-host

Something like 

```
RUN echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' > /etc/nsswitch.conf
```

Should match the mentioned fix in the upstream issue. Many thanks for the alpine sfj images!

**Comment by dinogun** [comment]
Thanks for reviewing the images ! I see the nsswitch.conf is already present and has the hosts entry. Did you run into an issue with Java not recognizing hosts ? (I regenerated the alpine images last week to include glibc 2.23-r2, so that might have pulled in the upstream fix)
/ # cat /etc/nsswitch.conf 
hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4

**Comment by dnwe** [comment]
Ah thanks. No was still using the image with 2.21-r2 and hadn't spotted ff3e526 had bumped the version

**Comment by dnwe** [comment]
Yep looks like sgerrard/alpine-pkg-glibc@6b1a253d2f1cb770bf86d80911e430130a31efc2 was included since the unreleased 2.22-r7 and hence was in the 2.23 that the images now pick up
